src/test/harness.js

```diff
@@ -374,7 +374,7 @@ async function executeInConsole(value) {
 function waitForFrameTimeline(width) {
   return waitUntil(() => {
     const elem = document.querySelector(".frame-timeline-progress");
-    return elem.style?.width == width;
+    return elem?.style?.width == width;
```

Comment:
looks like you only need one `?`

`elem?.style.width == width`

Reply:
fixed!